packages/htslib/default.nix

```diff
@@ -0,0 +1,70 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  zlib,
+  bzip2,
+  xz,
+  curl,
+  perl,
+}:
+
+stdenv.mkDerivation rec {
```

Comment:
You may want to switch to the `finalAttrs` pattern to fit the newer nixpkgs guidelines. Not necessary if upstreaming is not the goal here.